srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -1,68 +1,92 @@
+"""
+This script takes converts one realisation CSV file to an SRF output file.
+
+The parameters that are read from the CSV file are documented on the
+[wiki](https://wiki.canterbury.ac.nz/display/QuakeCore/File+Formats+Used+On+GM).
+"""
+
 import argparse
+import os
+import subprocess
+from subprocess import Popen
 from logging import Logger
-from subprocess import run, PIPE, Popen
-from typing import Dict, Any, Union, List
 from tempfile import NamedTemporaryFile
+from typing import Any, Dict, List, TextIO, Union, Tuple
 
-import yaml
-from h5py import File as h5open
 import numpy as np
-from os import makedirs, path, remove
-from qcore import binary_version, srf, geo, qclogging, utils
+import yaml
+import h5py
+from qcore import binary_version, geo, qclogging, srf, utils
+from qcore.uncertainties import mag_scaling
```

Comment:
Same here

Reply:
Yeah I can change this.

Reply:
Actually, I just tried this. Due to some weird import stuff only importing qcore and then trying to access the stuff under binary_version, geo, etc.. causes loads of errors. I think I'll add this to the "when I eventually rewrite this module" to-do list.